core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
@seanlip , I talked with Prasanna and Anmol regarding this. Prasanna thought that making this async right now won't be required. But, we can re-use this function when we build training jobs which doesn't need to be async. So, we can merge this to get the pipeline flowing. We can modify it later after build the job storage layer to work with the new framework. @anmolshkl , What are your thoughts?